commit de47f6323d8fb20feefee21d0195cf0529151e04
Author: Steve Dickson <steved@redhat.com>
Date:   Thu Sep 17 15:57:35 2015 -0400

    security.c: removed warning
    
    src/security.c:100:8: warning: implicit declaration of function 'xlog'
    [-Wimplicit-function-declaration]
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/src/security.c b/src/security.c
index 0c9453f..c54ce26 100644
--- a/src/security.c
+++ b/src/security.c
@@ -17,6 +17,8 @@
 #include <syslog.h>
 #include <netdb.h>
 
+#include "xlog.h"
+
 /*
  * XXX for special case checks in check_callit.
  */
